### DISCUSSION TITLE
Why do bookmarklets not work?

### DISCUSSION THREAD

**Discussion by Turbored80** [3kh0/ext-remover#1150] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't think that javascript is blocked but anytime I try to run a bookmarklet when it is in bookmark form it does not work does anyone know why? (I am not going to unenroll my Chromebook though), (Html files work), I'm trying to install extensions
![Screenshot 2024-08-29 3 26 59 PM](https://github.com/user-attachments/assets/6d1b7e7b-7fa8-4450-aeb5-2b6b1d8ce69e)

If you need a video to show you what is happening i can provide it. 

My blocker, Light speed, (I disable it with the Print exploit)

Chrome version: 123.0.6312.134 (Official Build) (64-bit)  

I have access to Crosh

If you need any extra info I can provide. 

Thank you for your help in advanced

**Comment by S-PScripts** [comment]
at some point bookmarklets were blocked on school chromebooks, even if it wasn't in the URL blocklist

**Reply by Turbored80** [reply]
But even on my home computer bookmarklets don't work on that profile but on my home account they do.

**Reply by AshtonDavies** [reply, marked as answer]
> But even on my home computer bookmarklets don't work on that profile but on my home account they do.

Because school profiles have policies apart from Chromebooks themselves, duh.

**Reply by Turbored80** [reply]
Yeah but I don't have JavaScript restricted on the allow list

**Reply by AshtonDavies** [reply]
> Yeah but I don't have JavaScript restricted on the allow list

@Turbored80 Patched v115 also.

**Reply by xoidex** [reply]
> Yeah but I don't have JavaScript restricted on the allow list

sometime during last year, bookmarkets were blocked by chrome (the browser) if the account was managed by some organization, eg. your school

**Reply by AshtonDavies** [reply]
> > Yeah but I don't have JavaScript restricted on the allow list
> 
> sometime during last year, bookmarkets were blocked by chrome (the browser) if the account was managed by some organization, eg. your school

Yes, it became set depending on devtools availability as it technically should be. This is common knowledge at this point.
